### DISCUSSION TITLE
GPU not be used in Sealing

### DISCUSSION THREAD

**Discussion by vsharecoder** [filecoin-project/lotus#6138] [Storage Provider] [answered]
After I updated my lotus-miner software to the latest version, my machine can not use GPU in PreCommit2 & Commit2 ...

**Comment by vsharecoder** [comment]
<img width="637" alt="2021-04-24 08 53 17" src="https://user-images.githubusercontent.com/79181863/115942132-435ea000-a4db-11eb-809e-f3aefb5041a0.png">

**Comment by jennijuju** [comment]
yes - there is small thing need to be fixed with proof so that gpu is used by default https://github.com/filecoin-project/filecoin-ffi/pull/162, but for now, if you build and run your lotus with `FFI_USE_GPU2=1` or `FFI_USE_GPU=1`, it will be loaded.

**Comment by jennijuju** [comment]
let me know if thats works.

**Comment by narkissos** [comment]
> let me know if thats works.

hi juju, I came across this issue as well. My understanding is we must export FFI_USE_GPU=1 before we build new lotus programs, is it? ...and is there any plan to address this in the official document? Thanks.

**Comment by narkissos** [comment]
> let me know if thats works.

Yes it works well after set FFI_USE_GPU=1. My question is, should we use FFI_USE_GPU=1 to enable GPU or just leave it disabled by default. Is there any benefit from disabling GPU in new version, or we should use GPU just like before, in order to get the max sealing/proofing speed?

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
no benefit. it's a "bug" that you need to set the flag. is resolved in one of the 1.9.x-rcY versions

**Comment by vsharecoder** [comment]
It works after I set FFI_USE_GPU=1

**Comment by SgtCoin** [comment, marked as answer]
Issue resolved, archiving for future reference.
